web/src/features/time/HistoricalTimeHeader.tsx

```diff
@@ -65,10 +82,14 @@ export default function HistoricalTimeHeader() {
             onClick={handleLeftClick}
             size="sm"
             type="tertiary"
+            isDisabled={!isWithinHistoricalLimit}
             icon={
               <ChevronLeft
                 size={22}
-                className={twMerge('text-brand-green', !isHourly && 'opacity-50')}
+                className={twMerge(
+                  'text-brand-green',
+                  !isWithinHistoricalLimit && 'opacity-50'
```

Comment:
Should this not become disabled on the last day/point inside the historical limit, not the first time outside it?

Reply:
I think it already does, I've updated the variable naming to try make this clearer